src/OVAL/probes/SEAP/seap-command-backendS.c

```diff
@@ -30,6 +30,7 @@
 #include "public/sm_alloc.h"
 #include "_seap-command.h"
 #include "seap-command-backendS.h"
+#include "seap-command-backendL.h"
```

Comment:
why? shouldnt' this be in maint-1.2 as well?

Reply:
@mpreisler I have already opened a PR on maint-1.2 with this. https://github.com/OpenSCAP/openscap/pull/901